src-docs/src/views/datagrid/column_cell_actions.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useCallback } from 'react';
+import { fake } from 'faker';
+
+import { EuiDataGrid, EuiAvatar } from '../../../../src/components/';
+import { EuiButton } from '../../../../src/components/button';
+
+const columns = [
+  {
+    id: 'avatar',
+    initialWidth: 40,
+    isResizable: false,
+    actions: false,
+  },
+  {
+    id: 'name',
+    initialWidth: 100,
+    isSortable: true,
+    actions: {
+      showHide: false,
+    },
+  },
+  {
+    id: 'email',
+    isSortable: true,
+    cellActions: [
+      {
+        iconType: 'heart',
+        label: 'Send love',
+        callback: (rowIndex, colIndex) =>
+          alert(
+            `Love letter sent to row "${rowIndex + 1}" column "${colIndex}"`
+          ),
+      },
+    ],
+  },
+  {
+    id: 'city',
+    isSortable: true,
+    cellActions: [
+      {
+        iconType: 'cheer',
+        label: 'Cheer the city',
+        callback: (rowIndex, columnId) =>
+          alert(`Cheers sent to row "${rowIndex + 1}" column "${columnId}"`),
+        inPopoverButton: (rowIndex, columnId) => (
+          <EuiButton
+            onClick={() =>
+              alert(
+                `Cheers sent in Popover to row "${rowIndex +
+                  1}" column "${columnId}"`
+              )
+            }>
+            Cheer
+          </EuiButton>
+        ),
+      },
+    ],
+  },
+  {
+    id: 'country',
+    cellActions: [
+      {
+        iconType: 'heart',
+        label: 'Love this city',
+        callback: (rowIndex, colIndex) =>
+          alert(`Love sent from row ${rowIndex + 1}, column "${colIndex}"`),
+      },
+      {
+        iconType: 'brush',
+        label: 'Paint this city',
+        callback: (rowIndex, colIndex) =>
+          alert(`Paint sent from ${rowIndex + 1}, column "${colIndex}"`),
```

Comment:
```suggestion
          alert(`Paint sent from row ${rowIndex + 1}, column "${colIndex}"`),
```
We're missing the word `row` here.